staging_docs/admin/guides/01-rbac.md

```diff
@@ -0,0 +1,379 @@
+# Role Based Access Control
+
+!!! warning
+    This feature is a Tech Preview and may be subject to change.
+
+
+!!! note
+    User `admin` automatically gets all RBAC permissions.
+
+
+RPM plugin presents Role Based Access for these endpoints :
+
+- RPM Alternate Content Source (`/pulp/api/v3/acs/rpm/rpm/`)
+- RPM Distributions (`/pulp/api/v3/distributions/rpm/rpm/`)
+- RPM Publications (`/pulp/api/v3/publications/rpm/rpm/`)
+- RPM Remotes (`/pulp/api/v3/remotes/rpm/rpm/` and `/pulp/api/v3/remotes/rpm/uln`)
```

Comment:
For consistency, should we have trailing slash on uln?

Reply:
For this pass, the rule is "DO NOT EDIT" :)  I've broken it a little, but generally trying not to go down the rabbit-hole of fixing All The Things. After we're in the New Format, we'll want a thorough editing pass - there is A LOT to fix up in here....